script/container-setup.sh

```diff
@@ -39,6 +39,7 @@ define DB_HOST db
 define DB_NAME ctfd
 define DB_USER ctfd
 define DB_PASS ctfd
+define DB_EXTERNAL no # change to anything but no and the db container will not start mysql
```

Comment:
My only real thought here is that `DB_EXTERNAL` is redundant with `DB_HOST` being vs not being `db`. As a result, `DB_HOST=db; DB_EXTERNAL=yes` would lead to a weird state.

I would personally lead towards less config options, but up to you if you prefer this explicit config option name.

Reply:
I didn't like the idea of further hardcoding in the infra of what the container name should be. 

In general I prefer explicit configuration vs. magic configuration. But it is redundant and there is a possibility of these weird states.